django/cantusdb_project/main_app/admin.py

```diff
@@ -65,7 +65,10 @@ class SegmentAdmin(BaseModelAdmin):
 
 
 class SequenceAdmin(BaseModelAdmin):
+    search_fields = ("title", "incipit", "cantus_id",)
     exclude = EXCLUDE + ("c_sequence", "next_chant", "is_last_chant_in_feast")
+    list_display = ("incipit", "siglum", "genre")
```

Comment:
> `"siglum"`

- `list_display` dictates which columns are displayed, yes? Do we want to use `"source__siglum"` instead of just `"siglum"`? (I'm still a little confused as to why chants/sequences have a "siglum" field in OldCantus, and I don't think we have anything set up to automatically set the siglum of new chants created in NewCantus, so I've been avoiding using BaseChant.siglum...)

Reply:
In our chant sync script we have
```    
try:
        siglum = json_response["field_siglum_chant"]["und"][0]["value"]
except (KeyError, TypeError):
        siglum = None
```

And the taxonomy manager does show each siglum that corresponds to the chant:
![image](https://github.com/DDMAL/CantusDB/assets/71031342/f0ff0891-e692-419c-bd4e-2fc80dea5a47)

However, if we are not using this field anywhere else in the code, it does make more sense to use source__siglum instead of siglum. (However, "source__siglum" doesn't behave as expected in the model admin, so we have to access it in a different way. Alternatively we could just use "source" which displays the full source __str__ representation.)